app.js

```diff
@@ -1,10 +1,53 @@
 const WebSocket = require('ws');
 require('dotenv').load();
-var exp = require('./index.js');
 
+// when socket opens, notify channel
+function onOpen (soc) {
+    console.log('socket opened');
+    var msg = {
+        "id": 1,
+        "type": "message",
+        "channel": "C1BBWJ7PF",
+        "text": "WhoopBot connected to WebSocket"
+    };
+    
+    if (soc.readyState == 1) {
+        soc.send(JSON.stringify(msg));
+    };
+};
 
+// log incoming messages
+function onMessage (event) {
+    var ev = JSON.parse(event);
+    var text = ev.text;
+    console.log(event);
+};
+
+
+// when HTTPS request finished, initialize WebSocket and handle events
+function finishedRequest(data) {
```

Comment:
I find "finishedRequest" a little confusing. I'd go with either "finishedRTStartRequest" (if you want to name the function for the reason that it's called) or "initializeWebsocket" (if you want to name the function for what it does). I prefer the latter approach.